pkg/component/apiserver/admission.go

```diff
@@ -206,3 +204,66 @@ func admissionPluginsConfigFilename(name string) string {
 func admissionPluginsKubeconfigFilename(name string) string {
 	return strings.ToLower(name) + "-kubeconfig.yaml"
 }
+
+// InjectAdmissionSettings injects the admission settings into the deployment.
```

Comment:
```suggestion
// InjectAdmissionSettings injects the admission settings into `gardener-apiserver` and `kube-apiserver` deployments.
```
In the context of this PR it is obvious that the function is used for `kube-apiserver` and `gardener-apiserver`.
I'm wondering if this is still the case if someone is changing this function in a year or so. Shall we mention both cases? The same question for the other `Inject...` functions.